### ISSUE TITLE
Remove ic_launcher icon from AndroidManifest

### ISSUE THREAD

**Issue by craigsailor** [wu9007/qrcode_scanner#120]
When trying to use qrscan, I get the following error:

Attribute application@icon value=(@mipmap/launcher_icon) from AndroidManifest.xml:11:6-42
	is also present at [:qrscan] AndroidManifest.xml:22:9-43 value=(@mipmap/ic_launcher).

I don't know why this package is including the application@icon directive in the AndroidManifest, but could you please remove it.

**Comment by wu9007** [comment]
@craigsailor agreed — a plugin must not declare `application@icon`. 0.4.0 plugin manifest no longer has it, and `ic_launcher` is not in the library resources.

Host `@mipmap/launcher_icon` should merge cleanly. Closing as completed.
